src/VirtoCommerce.Platform.Hangfire/Models/SettingCronJob.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Hangfire.States;
+using Hangfire.Common;
+using VirtoCommerce.Platform.Core.Settings;
+
+namespace VirtoCommerce.Platform.Hangfire
+{
+    public class SettingCronJob
```

Comment:
can we mark this class as internal?

Reply:
can't. only mark ctor as internal. is that enough?